Add docs to recreate peering token. (#14126)
hashicorp/consul: website/content/docs/connect/cluster-peering/k8s.mdx

```shell-session
$ curl "localhost:8500/v1/health/connect/backend?peer=cluster-02"
```

> ## Recreate/Reset a peering connection
> 
> To recreate or reset the peering connection, a new peering token needs to be generated on the cluster where the `PeeringAcceptor` was created, which in this case is `cluster-01`.
> 
> This can be performed by creating/updating the annotation `consul.hashicorp.com/peering-version` on the `PeeringAcceptor`. If the annotation already exists, update its value to a version that is higher.
> 
> Once the above is done, repeat the steps in the peering process from saving your peering token so that you can export it to the other cluster. This will re-establish peering with the updated token.
> 
> -> **NOTE:** A new peering token is only generated upon manually setting and updating the value of the annotation `consul.hashicorp.com/peering-version`. Creating a new token will cause the previous token to expire.

Reviewer: active voice. can we also provide an example config of what a `PeeringAcceptor` with the right `consul.hashicorp.com/peering-version` annotation looks like?

```suggestion
## Recreate or reset a peering connection

To recreate or reset the peering connection, you need to generate a new peering token on the cluster where you created the `PeeringAcceptor` (in this example, `cluster-01`(.

You can do this by creating or updating the annotation `consul.hashicorp.com/peering-version` on the `PeeringAcceptor`. If the annotation already exists, update its value to a version that is higher.

Once you have done this, repeat the steps in the peering process. This includes saving your peering token so that you can export it to the other cluster. This will re-establish peering with the updated token.

~> **Note:** A new peering token is only generated upon manually setting and updating the value of the annotation `consul.hashicorp.com/peering-version`. Creating a new token will cause the previous token to expire.
```

```shell-session
$ curl "localhost:8500/v1/health/connect/backend?peer=cluster-02"
```

## Recreate or reset a peering connection

To recreate or reset the peering connection, you need to generate a new peering token on the cluster where you created the `PeeringAcceptor` (in this example, `cluster-01`).

1. You can do this by creating or updating the annotation `consul.hashicorp.com/peering-version` on the `PeeringAcceptor`. If the annotation already exists, update its value to a version that is higher.

>     <CodeBlockConfig filename="acceptor.yml">

Reviewer: ```suggestion
    <CodeBlockConfig filename="acceptor.yml" highlight="6" hideClipboard>
```
